wiki/Accuracy/en.md

```diff
@@ -8,11 +8,11 @@ Accuracy is a percentile measurement of a player's ability to hit [hit objects](
 
 ## Game modes
 
-### ![](/wiki/shared/mode/osu.png) osu!standard
+### ![](/wiki/shared/mode/osu.png) osu! mode
 
-![Accuracy = (50 \* number of 50s + 100 \* number of 100s + 300 \* number of 300s) / 300(number of 0s + number of 50s + number of 100s + number of 300s)](img/accuracy_standard.png "Accuracy formula for osu!standard")
```

Comment:
regarding the file name, are there any other files with exclamation mark in name on osu! wiki at all? if not, I'd be cautious and leave this as `accuracy_osu.png`